data/type.go

```diff
@@ -0,0 +1,27 @@
+package data
+
+type FieldsGetter interface {
```

Comment:
do we use these interfaces?

Reply:
yes, I am using them in the [tests](https://github.com/multiversx/mx-chain-ws-connector-firehose-go/blob/88e956a361d98c4a02f3f774857d19f2d9b1c77b/process/outportBlockConverter_test.go).

Reply:
ok, please add missing comments